ern-api-gen/src/generateProject.js

```diff
@@ -92,14 +93,20 @@ export function generatePackageJson ({
   }, null, 2)
 }
 
-export function generateInitialSchema ({
+export async function generateInitialSchema ({
```

Comment:
Why making it `async` ? It does not await on anything nor return a Promise.

Reply:
@belemaire fileUtils.readFile is a async function

Reply:
My bad ;)